_schemaV2.graphql

```diff
@@ -2163,20 +2163,16 @@ type ArtistRelatedData {
   ): ArtistConnection
 }
 
-type ArtistSeries {
```

Comment:
This file was generated with the flag **enabled** - I'll undo before merging, this is more of a preview of how the schema looks when we toggle the flag on.

Reply:
On second thought - b/c everything is ready to go in this PR, _and_ there shouldn't be any breaking changes to clients, it might be good to even merge this (and generate the schema w/ the flag enabled - ordinarily we wouldn't do that yet but since this is all done in one PR, might be fine and will get a preview of any client issues with accepting this new schema)